dashboard/test/fixtures/census/other_curriculum_offerings.csv

```diff
@@ -0,0 +1,408 @@
+School ID,Course,Year,School Name,City,Zip/Postal Code,School District,State/Province
```

Comment:
Is this data used by a test?

Reply:
If locals.yml `stub_school_data` is `true` (most dev environments, this file is imported instead of importing all of the files S3.